contracts/destroyable/Destroyable.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.6.0;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Destroyable is Ownable {
+    function kill() public onlyOwner {
```

Comment:
We probably don't need to have this method if it is that simple. We might be able to use a delegatecall to a contract that implements a selfdestruct call.